### PR TITLE
[12.0][MIG] web_action_conditionable

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@ageficent double check please

This module is used for the sale_blanket_order and the purchase_blanket_order.

File: web_action_conditionable/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Not needed.
```
Comment (alexey-pelykh) [inline web_action_conditionable/readme/CONFIGURE.rst]: No need for this file then?

File: web_action_conditionable/readme/INSTALL.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+To install this module, just follow basic steps to install an odoo module.
```
Comment (alexey-pelykh) [inline web_action_conditionable/readme/INSTALL.rst]: No specific steps, this file is not needed?

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Minor documentation tweaks needed

File: web_action_conditionable/readme/CONFIGURE.rst (hunk 1 above)
Comment (nikul-serpentcs) [reply web_action_conditionable/readme/CONFIGURE.rst]: @alexey-pelykh then Remove it :smile:

Comment (alexey-pelykh) [reply web_action_conditionable/readme/CONFIGURE.rst]: well, I don't think I can :) It's @mreficent PR after all

Comment (MiquelRForgeFlow) [reply web_action_conditionable/readme/CONFIGURE.rst]: done

**Review by alexey-pelykh — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]
Code review LGTM :+1:

**Pull request closed without merging**